src/perf.js

```diff
@@ -8,6 +8,7 @@ import { stringify } from 'query-string';
 import { median, quantile } from 'simple-statistics';
 import { getEvolution, getLatestEvolution } from './perf/tmo';
 import { fetchTelemetryEvolution } from './perf/tmo-wrapper';
+import fetchTransformSpeedometerData from './perf/speedometer';
```

Comment:
to be pedantic I think this should be two lines up (to keep things in alphabetical order)